hera_cal/delay_filter.py

```diff
@@ -128,13 +128,21 @@ def load_delay_filter_and_write(infilename, calfile=None, Nbls_per_load=None, sp
                                clobber=clobber, add_to_history=add_to_history,
                                extra_attrs={'Nfreqs': len(freqs), 'freq_array': np.asarray([freqs])})
     else:
+        nwrites = 0
```

Comment:
I guess if we are comparing this to a waterfall with a large number of times, then the impact on writes is trivial. If we are processing a small number of times and baselines per partial i/o then it could have a large impact.

Reply:
I think it's a trivial effect on i/o regardless, since `freq_array` is small compared to any waterfall. I'd err on the side of simpler code.